build.gradle

```diff
@@ -65,6 +66,16 @@ dependencies {
 
 }
 
+task javadocStrict(type: Javadoc) {
+  group = 'Documentation'
+  description = 'Run Javadoc in strict mode: with -Xdoclint:all and -Xwerror, on all members.'
+  source = sourceSets.main.allJava
+  classpath = sourceSets.main.runtimeClasspath
+  options.addStringOption('Xwerror', '-Xdoclint:all')
+  options.memberLevel = JavadocMemberLevel.PACKAGE
+}
+check.dependsOn javadocStrict
```

Comment:
Temporary to enable faster debugging with "gradle check" syntax.  Will change to run with "gradle test".

Reply:
Actually, gradle check *is* run on the PR so this is good.